x-pack/test/functional/apps/ml/data_frame_analytics/cloning.ts

```diff
@@ -16,7 +16,7 @@ export default function ({ getService }: FtrProviderContext) {
   const ml = getService('ml');
 
   // Failing ES promotion, see https://github.com/elastic/kibana/issues/89980
-  describe.skip('jobs cloning supported by UI form', function () {
+  describe('jobs cloning supported by UI form', function () {
```

Comment:
Would be good to also remove the comment that's related to the `skip`.

Reply:
Good catch - removed in e4f892e9bce90a0e01b7f7c3183a383c055ad4d7